tox.ini

```
@@ -1,44 +1,66 @@
 [tox]
 minversion = 3.7.0
```

let's bump to have the newest bugfixes and a few nice improvements..
```suggestion
minversion = 3.8.1
```

---

Adding a requirement on tox version that was released 60 minutes ago is not safe.

Let things rest before bumping requirements to bleading edge releases.

---

I think there's no reason to distrust tox's release standards. Unless there's a specific problem that comes from using 3.8, we should feel free to track the latest tox. tox claims to follow semantic versioning, which we also have no reason to distrust. I'm not generally in favor of upgrading for upgrading's sake, but improvements to parallel testing in 3.8 would likely help get `tox .. -p auto` working.

---

Though, in line with the idea of incremental changes, it makes sense to only bump the tox minimums if tests are updated to require features introduced in higher versions.

---

It's only a patch release which fixed a minor bump. It doesn't matter when it was released, only proves that fixes get released fast.
And yes, tox's standards are high and current maintainers are very responsible.
The reason I want this update is that it ships certain features and bugfixes which I'd like to rely on.